src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -0,0 +1,384 @@
+"""add default policies
+
+Revision ID: 55d61eb8ed12
+Revises: b3b68c87c4a0
+Create Date: 2022-06-13 19:26:24.197262
+
+"""
+import logging
+from typing import Optional, Tuple
+from uuid import uuid4
+
+from alembic import op
+from sqlalchemy import text
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.engine import LegacyRow
+from sqlalchemy.engine.base import Connection
+from sqlalchemy.sql.elements import TextClause
+from sqlalchemy_utils import StringEncryptedType
+from sqlalchemy_utils.types.encrypted.encrypted_type import AesGcmEngine
+
+from fidesops.api.v1.scope_registry import SCOPE_REGISTRY
+from fidesops.core.config import config
+from fidesops.db.base import Policy, Rule, RuleTarget, StorageConfig
+from fidesops.db.base_class import FidesopsBase, JSONTypeOverride
+from fidesops.models.policy import ActionType, DrpAction
+from fidesops.schemas.storage.storage import StorageType
+from fidesops.service.masking.strategy.masking_strategy_string_rewrite import (
+    STRING_REWRITE_STRATEGY_NAME,
+)
+from fidesops.util.cryptographic_util import (
+    generate_salt,
+    generate_secure_random_string,
+    hash_with_salt,
+)
+from fidesops.util.data_category import DataCategory
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+
+revision = "55d61eb8ed12"
+down_revision = "b3b68c87c4a0"
+branch_labels = None
+depends_on = None
+
+FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fidesops_autogenerated_client"
+FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+AUTOGENERATED_ACCESS_KEY = "download"
+AUTOGENERATED_ERASURE_KEY = "delete"
+
+client_select_query: TextClause = text(
+    """SELECT client.id FROM client WHERE fides_key = :fides_key"""
+)
+
+client_insert_query: TextClause = text(
+    """
+    INSERT INTO client (id, hashed_secret, salt, scopes, fides_key) 
+    VALUES (:client_id, :hashed_secret, :salt, :scopes, :fides_key)
+    """
+)
+
+client_delete_query: TextClause = text(
+    """DELETE FROM client WHERE client.id = :client_id"""
+)
+
+storage_select_query: TextClause = text(
+    """SELECT storageconfig.id FROM storageconfig WHERE storageconfig.key = :storage_config_key"""
+)
+storage_insert_query: TextClause = text(
+    """
+    INSERT INTO storageconfig(id, name, type, details, key, secrets, format) 
+    VALUES(:storage_config_id, :storage_name, :storage_type, '{"naming": "request_id"}', 
+    :storage_key, NULL, 'json')
+    """
```

Comment:
An earlier commit was using the SQLAlchemy ORM to run these queries, but this will break in the future as these models are updated, and we're importing the most recent model into older migrations where the columns don't exist yet.  These migrations should be able to stand on their own.

Reply:
Have you tested that this is the case explicitly? As far as I'm aware this wouldn't be an issue since the migration is run at a specific point in the migration history, meaning we can guarantee the DB state at that point based on what's run before it.

Reply:
> and then you have the latest version of the SQLAlchemy models trying to do things in this older migration that isn't possible yet.

Got it, that makes sense!